.github/actions/install-dependencies/action.yml

```diff
@@ -0,0 +1,27 @@
+name: Install Dependencies
+description: Installs system dependencies
+
+inputs:
+  os:
+    description: Supports Linux and macOS
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install system dependencies (Linux)
+      if: inputs.os == 'Linux'
```

Comment:
Why not use `runner.os` directly?

Reply:
I thought actions do not have access to `runner`. Used it in new commit.